sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -0,0 +1,546 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# TODO:
+# - add USE flag for remote modules? Those modules can be downloaded
+#	properly before building.
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+WEBAPP_OPTIONAL=yes
+WEBAPP_MANUAL_SLOT=yes
+
+# qmake-utils
+inherit check-reqs cmake cuda flag-o-matic java-pkg-opt-2 python-single-r1 toolchain-funcs virtualx webapp
+
+# Short package version
+SPV="$(ver_cut 1-2)"
+
+# needed for testing
+MISSING_TEST_FILES=(
+	eaab2cdba68e8630f9a1f8a8e81cb097
+	325acbb1a74ae27148d76399c2e7c7c5
+	8e09d028a9fd114b4ceb8f924d8b466c
+	7e7e6c5e28fa93b16fe0414881bdbbcd
+)
+
+DESCRIPTION="The Visualization Toolkit"
+HOMEPAGE="https://www.vtk.org/"
+SRC_URI="
+	https://www.vtk.org/files/release/${SPV}/VTK-${PV}.tar.gz
+	https://www.vtk.org/files/release/${SPV}/VTKData-${PV}.tar.gz
+	doc? ( https://www.vtk.org/files/release/${SPV}/vtkDocHtml-${PV}.tar.gz )
+	examples? ( https://www.vtk.org/files/release/${SPV}/VTKLargeData-${PV}.tar.gz )
+	test? (
+		https://www.vtk.org/files/ExternalData/MD5/eaab2cdba68e8630f9a1f8a8e81cb097
+		https://www.vtk.org/files/ExternalData/MD5/325acbb1a74ae27148d76399c2e7c7c5
+		https://www.vtk.org/files/ExternalData/MD5/8e09d028a9fd114b4ceb8f924d8b466c
+		https://www.vtk.org/files/ExternalData/MD5/7e7e6c5e28fa93b16fe0414881bdbbcd
+		https://www.vtk.org/files/release/${SPV}/VTKLargeData-${PV}.tar.gz
+	)
+"
+
+LICENSE="BSD LGPL-2"
+# several dependencies need to be keyworded for arm
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+SLOT="0"
+# New in 9.0?: sdl -> from which group / module is it coming?
```

Comment:
TODO

Reply:
I saw this dep once, maybe on a try from the git branch. Will check, if I can reconstruct this, or else remove the comment.

Reply:
Couldn't reproduce where I saw this and can not find it by grepping the .cmake and CMakeLists.txt files. I ignore this for now.